README.md

```diff
@@ -1,6 +1,8 @@
 # Reddit Enhancement Suite
 
 [![Build Status](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite.svg?branch=master)](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite)
+[![Build status](https://ci.appveyor.com/api/projects/status/ah6r3ij1yqtkc0ad/branch/master?svg=true)](https://ci.appveyor.com/project/erikdesjardins/reddit-enhancement-suite-md1r4/branch/master)
```

Comment:
Interesting, seems that Appveyor projects are per-user, not per-repo. It's definitely building this repo though: https://ci.appveyor.com/project/erikdesjardins/reddit-enhancement-suite-md1r4/build/1/job/3v0yu95bd1o25ou3#L2